### PR TITLE
Player to console

### PR DESCRIPTION
**Pull request by CholiCub**
Overloading << operator for Player console to satisfy Dependency inversion

### CODE REVIEW

File: AJourenyForLove-Text-BasedAdventureGame/Date.cpp [hunk 1]
```diff
@@ -0,0 +1 @@
+#include "Date.h"
```
Comment (Rosuavio) [inline AJourenyForLove-Text-BasedAdventureGame/Date.cpp]: It does not seem like you need this file.

File: AJourenyForLove-Text-BasedAdventureGame/Player.cpp [hunk 2]
```diff
@@ -68,4 +47,24 @@ EEyeColor Player::getEyeColor() const {
...
+	PartnerPreference pp;
+	os << "Name: " << p.name <<"\n";
+	os << "Age: " << p.age << "\n";
+	os << "Identity: " << i.ToString(p.identity) << "\n";
+	os << "Race: " << ra.ToString(p.race) << "\n";
+	os << "Religion: " << re.ToString(p.religion) << "\n";
+	os << "Body Type: " << bt.ToString(p.bodyType) << "\n";
+	os << "Hair Color: " << hc.ToString(p.hairColor) << "\n";
+	os << "Eye Color: " << ec.ToString(p.eyeColor) << "\n";
+	os << "Partner Preference: " << pp.ToString(p.partnerPreference) << "\n";
```
Comment (Rosuavio) [inline AJourenyForLove-Text-BasedAdventureGame/Player.cpp:68]: I think that `std::endl` would be automatically compatible with more systems over `"\n"`.

**Review by Rosuavio — CHANGES_REQUESTED** [review]
Looks really good, one change I would recommend.

File: AJourenyForLove-Text-BasedAdventureGame/Player.cpp (hunk 2 above)
Comment (Rosuavio) [reply AJourenyForLove-Text-BasedAdventureGame/Player.cpp:68]: Also it seems you were using `endl` before too so might as well keep it.

Comment (CholiCub) [reply AJourenyForLove-Text-BasedAdventureGame/Player.cpp:68]: Okay, will change

File: AJourenyForLove-Text-BasedAdventureGame/Date.cpp (hunk 1 above)
Comment (CholiCub) [reply AJourenyForLove-Text-BasedAdventureGame/Date.cpp]: the change no longer in branch

File: AJourenyForLove-Text-BasedAdventureGame/Player.cpp (hunk 2 above)
Comment (CholiCub) [reply AJourenyForLove-Text-BasedAdventureGame/Player.cpp]: 5fb3c0bb440a79e438c2c9222f1cf928c2a4d232

**Review by Rosuavio — APPROVED** [review]
Looks good :+1:

**Pull request merged**